packaging/wix/configure.py

```diff
@@ -0,0 +1,18 @@
+from os import path
+from string import Template
+import toml
+
+def get_version():
+    with open(path.join("..", "..", "Cargo.toml")) as f:
+        cargo = toml.load(f)
+        return cargo["package"]["version"]
+
+def configure_wxs_file(version):
+    with open("indentex_template.wxs") as f:
+        tmpl = Template(f.read())
+
+    with open("indentex.wxs", "w") as f:
+        f.write(tmpl.substitute(version=version))
```

Comment:
Maybe it would be better to pass a context? Something along the lines of `tmpl.substitute(**context)`.

It's more future-proof if someday we have more variables than just the version.

However, don't write any code you don't need right now, so...